pkg/log/tty.go

```diff
@@ -100,57 +100,75 @@ func (*TTYWriter) Fatalf(format string, args ...interface{}) {
 // Green writes a line in green
 func (w *TTYWriter) Green(format string, args ...interface{}) {
 	log.out.Infof(format, args...)
+	holdSpinner()
 	w.FPrintln(w.out.Out, greenString(format, args...))
+	unholdSpinner()
 }
 
 // Yellow writes a line in yellow
 func (w *TTYWriter) Yellow(format string, args ...interface{}) {
 	log.out.Infof(format, args...)
+	holdSpinner()
 	w.FPrintln(w.out.Out, yellowString(format, args...))
+	unholdSpinner()
 }
 
 // Success prints a message with the success symbol first, and the text in green
 func (w *TTYWriter) Success(format string, args ...interface{}) {
 	log.out.Infof(format, args...)
+	holdSpinner()
 	w.Fprintf(w.out.Out, "%s %s\n", coloredSuccessSymbol, greenString(format, args...))
+	unholdSpinner()
 }
 
 // Information prints a message with the information symbol first, and the text in blue
 func (w *TTYWriter) Information(format string, args ...interface{}) {
 	log.out.Infof(format, args...)
```

Comment:
Checking out https://github.com/okteto/okteto/blob/0950b15bc123fe1a6c5567f333c50053abb93855/pkg/log/format.go#L62 , I see that we should have `log.out` set inside `w.out`. I know that might be out of scope, so feel free to create another issue for that. And I think we might need to do something similar with the `spinner`